### ISSUE TITLE
installation failing python 2.7.10 and pip 10.0.0

### ISSUE THREAD

**Issue by javierarilos** [Nekmo/gkeep#4]
Hello there,

I think `gkeep` is a very cool project, thx!

I am trying to install it but there is an error:
```bash
      File "/private/var/folders/y5/0dfhk3_s1k94ql1x_4mm0lhm0000jh/T/pip-install-Yc2fc4/gkeep/setup.py", line 6, in <module>
        from pip.req import parse_requirements
    ImportError: No module named req
```

After googling, it seems that pip.req was an interal pip module and is not available anymore: 

https://github.com/pypa/pip/issues/3240

Software versions in my system:
* Python 2.7.10
* pip 10.0.0

```bash
➜  ~ python --version
Python 2.7.10
➜  ~ pip --version
pip 10.0.0 from /Library/Python/2.7/site-packages/pip (python 2.7)
➜  ~ python
Python 2.7.10 (default, Feb  7 2017, 00:08:15)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.34)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from pip.req import parse_requirements
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named req
>>>
```

**Comment by Nekmo** [comment]
Pip has upgraded to Pip 10 and is now incompatible. I'm fixing the problem.

**Comment by Nekmo** [comment]
It's fixed. Thank you very much for your issue!

It is already fixed in version 0.1.1.

**Comment by javierarilos** [comment]
👏 👏 👏 👏 👏 
thank you!
